src/login/login.component.tsx

```diff
@@ -13,7 +13,7 @@ interface LoginReferrer {
 }
 
 interface LoginProps
-  extends RouteComponentProps<{}, StaticContext, LoginReferrer> {}
+  extends RouteComponentProps<{}, undefined, LoginReferrer> {}
```

Comment:
What's this about?

Reply:
?

Reply:
This type (`StaticContext`) does not matter - unfortunately its not re-exported from `react-router-dom`, but I don't want (even for types) mention `react-router`.